tools/scripts/authz/dao_genesis_authorizations.json

```diff
@@ -97,5 +97,23 @@
       "msg": "\/poktroll.shared.MsgUpdateParam"
     },
     "expiration": "2500-01-01T00:00:00Z"
+  },
+  {
+    "granter": "pokt10d07y265gmmuvt4z0w9aw880jnsr700j8yv32t",
+    "grantee": "pokt1eeeksh2tvkh7wzmfrljnhw4wrhs55lcuvmekkw",
+    "authorization": {
+      "@type": "\/cosmos.authz.v1beta1.GenericAuthorization",
+      "msg": "\/cosmos.upgrade.v1beta1.MsgSoftwareUpgrade"
```

Comment:
THE magic line

Reply:
Is there a way we can leave a comment somehow or call it out? I know it's JSON so curious what ideas you have.

Reply:
@Olshansk with more grants introduced in #717 I think we need a separate document describing the available parameters and transactions.. There are too many. Separate small ticket?

Also, I've noticed some of our own parameters were removed but traces (tests, makefile target, etc) still present (for example `min_relay_difficulty_bits` - I assume there could be others). It can be tackled on the same ticket.